content/doc/pipeline/index.adoc

```diff
@@ -599,14 +599,17 @@ a build tool):
 [source, groovy]
 ----
  node {
-   git url: 'https://github.com/joe_user/simple-maven-project-with-tests.git'
+   git 'https://github.com/joe_user/simple-maven-project-with-tests.git'
    def mvnHome = tool 'M3'
    sh "${mvnHome}/bin/mvn -B -Dmaven.test.failure.ignore verify"
-   step([$class: 'ArtifactArchiver', artifacts: '**/target/*.jar', fingerprint: true])
-   step([$class: 'JUnitResultArchiver', testResults: '**/target/surefire-reports/TEST-*.xml'])
+   archiveArtifacts artifacts: '**/target/*.jar', fingerprint: true
```

Comment:
For my own edification, does this mean that `archive` is no longer supported or does it just not have the same options (`fingerprint: true`) as `archiveArtifact`?